docs/content/guides/developer/getting-started/sui-install.mdx

```diff
@@ -29,28 +29,91 @@ brew install mystenlabs/tap/sui
 
 Each Sui release provides a set of binaries for several operating systems. You can download these binaries from GitHub and use them to install Sui.
 
+<Tabs groupId="operating-systems">
+
+<TabItem value="linux" label="Linux">
 1. Go to https://github.com/MystenLabs/sui.
 1. In the right pane, find the **Releases** section.
 
     ![Sui releases in GitHub](./images/releases.png)
 1. Click the release tagged **Latest** to open the release's page.
 1. In the **Assets** section of the release, select the .tgz compressed file that corresponds to your operating system.
 1. Double-click the downloaded file. If the file doesn't automatically expand, manually unzip the file.
-1. Open the expanded folder and double-click the appropriate binary to install, beginning with sui-`<OS>`-`<ARCHITECTURE>`:
+1. Open the expanded folder. Navigate to the `target/release` directory and double-click the appropriate binary to install, beginning with `sui-ubuntu-x86_64`:
 
-    - sui-faucet-`<OS>`-`<ARCHITECTURE>`: Local faucet to mint coins on local network.
-    - sui-indexer-`<OS>`-`<ARCHITECTURE>`: An indexer for a local Sui network.
-    - sui-`<OS>`-`<ARCHITECTURE>`: Main Sui binary.
-    - sui-node-`<OS>`-`<ARCHITECTURE>`: Run a local node.
-    - sui-test-validator-`<OS>`-`<ARCHITECTURE>`: Run test validators on a local network for development.
-    - sui-tool-`<OS>`-`<ARCHITECTURE>`: Provides utilities for Sui.
+    - `sui-data-ingestion-ubuntu-x86_64`: Massage Full node data for indexer to store in a database.
+    - `sui-faucet-ubuntu-x86_64`: Local faucet to mint coins on local network.
+    - `sui-indexer-ubuntu-x86_64`: An indexer for a local Sui network.
+    - `sui-ubuntu-x86_64`: Main Sui binary.
+    - `sui-node-ubuntu-x86_64`: Run a local node.
+    - `sui-test-validator-ubuntu-x86_64`: Run test validators on a local network for development.
+    - `sui-tool-ubuntu-x86_64`: Provides utilities for Sui.
 
     The Sui binary takes several minutes to download and install files, so make sure you allocate enough time for it to complete. Actual time to complete depends on your network connection and computer specifications.
```

Comment:
Is this the Sui node binary? AFAICT none of these binaries when run will download or install anything.

Reply:
Binary equals executable so not required to download or install.